### DISCUSSION TITLE
自定义短语，为什么有的有效，有的无效？

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1719] [Q&A] [answered]
在custom_phrase.txt文件里写：

噷	oh
呣	om
呒	om
嗯  on

fT3 ftsd
fT4 ftsi
rT3 rtsd

GnRH    gnrh
IFNγ    ifng
IGF-1   igfi
NF-κB	nfkb

Neu5Ac	neua
Neu5Gc	neug

ω-3	omsd
ω-6	omlj
ω-7	omqi
ω-9	omjj

pH	ph
μg	wwke
mg/100g	mgbg

β羟基丁酸	qgjidqsr
N-乙酰半胱氨酸	yixnbdggodsr

结果，NF-κB之前的都不生效，而之后的都生效。。
这会是什么原因呢？
[custom_phrase.txt](https://github.com/user-attachments/files/17853552/custom_phrase.txt)

**Comment by lotem** [comment, marked as answer]
用 Tab 字符分隔字段。

**Reply by kwindva** [reply]
的确是用tab来分隔的，但是在VS Code里面，按tab之后，实际上出现的还是很多空格好像。。。
我看原本的那个yaml文件里，也都是这样的，这样会有问题吗？

**Reply by kwindva** [reply]
果然是这个问题。。。
我搜了下网络，在VSCode里面，把Editor: Insert Spaces禁掉，就能输入真正的tab而非多个空格了。。然后就真的能用了。。
非常感谢。
